src/helpers/findAll.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactTestInstance } from 'react-test-renderer';
+import { getConfig } from '../config';
+import { BaseOptions } from '../queries/options';
+import { isInaccessible } from './accessiblity';
+
```

Comment:
For clarity I think we should define 
```ts
interface FindAllOptions {
  hidden?: boolean;
}
```
which would be used in `findAll(..., options: FindAllOptions)`.

This will intentionally overlap with BaseQueryOptions, so that TS will be happy.

Reply:
agreed, but then findAll should be used in all queries by default, so maybe we should have CommonQueryOptions extends FindAllOptions right?

Reply:
Yeah, makes sens for CommonQueryOptions to extend FindAllOptions